.github/workflows/build.yml

```diff
@@ -37,15 +38,29 @@ jobs:
             --cache-from=$CACHE_FROM:latest \
             --tag artifact
 
-      - name: Push image
+          docker build infra/object-cache \
+            --file infra/object-cache/Dockerfile \
+            --tag object-cache
+
+          docker build infra/page-cache \
+            --file infra/page-cache/Dockerfile \
+            --tag page-cache
```

Comment:
All this should be shoved into the make file in issue #52 . But a separate pr for a different time.